figures/drawing-slice-2.fig

```diff
@@ -0,0 +1,36 @@
+                             AS 1000   ◄────BGP───► AS 65000
+
+ ┌───┐                     ┌────────┐                    ┌───┐
+ │NF1│       172.30.1/30   │        │   172.30.2/30      │NF2│
```

Comment:
Please use addresses from https://datatracker.ietf.org/doc/rfc5737/. Thanks.

Reply:
Is there any convention for AS path - using here a public AS 1000 for the example - ?

Reply:
Yes: 65536 - 65551 as per https://www.rfc-editor.org/rfc/rfc5398.html